src/runtime/msg/internal.rs

```diff
@@ -118,4 +118,8 @@ pub enum Internal {
     SeekLogsResult(SeekLogRequest, Result<SuccessResponse, CtxError>),
     /// The result of querying the data for LocalSearch
     LoadLocalSearchResult(Url, Result<Vec<Searchable>, EnvError>),
+    /// Result for getModal request
+    GetModalResult(APIRequest, Result<GetModalResponse, CtxError>),
+    /// Result for getNotification request
+    GetNotificationResult(APIRequest, Result<GetNotificationResponse, CtxError>),
```

Comment:
Can we have `null` responses when there is no Modal or Notification?

Reply:
It should not, the api respond with this pattern: 
```
{ result: ..., error: ... }
```